notification-portlet-webapp/src/main/webapp/scripts/modal.js

```diff
@@ -0,0 +1,129 @@
+Vue.component("modal", {
+  template: "#modal-template",
+  props: {
+    id: String
+  }
+})
+
+// start app
+new Vue({
+  el: "#notifications-vue-app",
```

Comment:
👍 Thanks @bpowell for the new id!

Reply:
I'm wondering if we would want to export this as a web component.
https://github.com/karol-f/vue-custom-element